src/Arcus.Templates.WebApi/appsettings.json

```diff
@@ -2,4 +2,10 @@
   //#if (CertificateAuth)
   "CertificateSubject": "YOUR CERTIFICATE SUBJECT NAME"
   //#endif
+  //#if (CertificateAuth && Serilog)
+  ,
+  //#endif
+  //#if (Serilog)
```

Comment:
Maybe something like `telemetry:applicationInsights:key` would be more structurized?

Reply:
:+1:

Reply:
It's a shame that I require to make a special condition for that comma 😒 
If we would have a value in the appsettings that is always there; then each if/endif section could include the comma and there would be no problem; but since we haven't that(?) it have to be like this.

Or should I just put in a test value or something? For demostration purposes, but also to help making this conditioning easier?

Reply:
I'm not sure what you are talking about, can you give an example please?

Reply:
Because we use an empty appsettings, we get the problem when we use project options that require an appsetting. Why? Because of the commas between the values. Only when (in this case) both project options are selected, we require a comma. 
This problem or inconvenience could be fixed if we would add a dummy test value to the appsettings. For demonstration purposes or so. Then each if/endif section in the file could include a trailing comma.

I'll look for a better approach with maybe some replacement logic but other than that sis adding a dummy demonstration value the easiest solution.

Reply:
Oh that's ok then, let's just move on.

Reply:
Ok, done.